astropy/modeling/core.py

```diff
@@ -2315,6 +2407,113 @@ def _custom_model_wrapper(func, fit_deriv=None):
     return type(model_name, (FittableModel,), members)
 
 
+def render_model(model, arr=None, coords=None, copy=True):
+    """
+    Evaluates a model on an input array. Evaluation is limited to
+    a bounding box if the `Model.bounding_box` attribute is set.
+
+    Parameters
+    ----------
+    model : `Model`
+        Model to be evaluated.
+    arr : `numpy.ndarray`, optional
+        Array on which the model is evaluated.
+    coords : array-like, optional
+        Coordinate arrays mapping to ``array``, such that 
+        ``arr[coords] == arr``.
+    copy : bool, optional
+        Default is ``True``. If ``False``, ``arr`` will be modified in place. 
+
+    Returns
+    -------
+    array : `numpy.ndarray`
+        The model evaluated on the input ``arr`` or a new array from ``coords``.  
+        If ``arr`` and ``coords`` are both `None`, the returned array is 
+        limited to the `Model.bounding_box` limits. If 
+        `Model.bounding_box` is `None`, ``arr`` or ``coords`` must be passed.
+    """
+
+    bbox = model.bounding_box
+
+    if (coords is None) & (arr is None) & (bbox is None):
+        raise AssertionError('If no bounding_box is set, coords or arr must be input.')
+
+    # for consistent indexing
+    if model.n_inputs == 1:
+        if coords is not None:
+            coords = [coords]
+        if bbox is not None:
+            bbox = [bbox]
+
+    if arr is not None:
+        # Check dimensions match model
+        assert arr.ndim == model.n_inputs
+        # Copy
```

Comment:
Remove the `# Copy` comment ... it doesn't add extra info over what the code says.